.github/workflows/ci.yml

```diff
@@ -24,7 +28,7 @@ jobs:
       - run: yarn install --immutable
       - run: yarn check-dependencies
 
-      - run: yarn nx affected --target=lint --parallel --max-parallel=3
-      - run: yarn nx affected --target=type-check --parallel --max-parallel=3
-      - run: yarn nx affected --target=build --parallel --max-parallel=3
-      - run: yarn nx affected --target=test --parallel --max-parallel=2
+      - run: yarn nx affected --target=lint
```

Comment:
question:

1. should we enable bail mode ? with this state if 1 task failed everything else will still run taking addional resources. I'd suggested to use `--nxBail`

2. wondering if we can enable all in one mode to utilize better resources  `yarn nx affected -t lint,type-check,build,test` ?

Reply:
1 & 2, sure, nice idea 👍